src/color_laws/fitzpatrick.jl

```diff
@@ -262,7 +262,9 @@ and Rv. For more information, seek the original paper.
 """
 function f19_invum(x::Real, Rv::Real)
     # read and unpack tabulated data
-    data_x, data_k, data_delta_k, data_sigma_k = let data = readdlm(joinpath(datadep"F19", "F19_tabulated.dat"), skipstart=1)
+    # using type annotations so `interpolate` can be inferred
+    # TODO: Avoid using readdlm to avoid this issue altogether
+    data_x::Vector{Float64}, data_k::Vector{Float64}, data_delta_k::Vector{Float64}, data_sigma_k::Vector{Float64} = let data = readdlm(joinpath(datadep"F19", "F19_tabulated.dat"), skipstart=1)
         (data[:, i] for i in 1:4)
     end
```

Comment:
With the caveat that I'm not familiar with this code, it seems to me it is inefficient to re-read the data for every call to `f19_invum`.

Why don't we load it as a constant (something like `const f19_data`) at the module level? I guess that would force download the datadep when you load the package but that sounds fine to me. That would also allow it's type to be known.

Frankly the data file's not even that large so we could just copy-paste the raw data into the source file unless there's some licensing issue or something ...

Reply:
100% agree. I was very tempted to just inline it while writing out that TODO. Will play around with some schemes later tn